R/models.R

```diff
@@ -14,6 +14,7 @@ basic_model <- function(progression_groups = list(E = 2, asympt = 1, mild = 1, I
   model_class <- "sircovid_basic"
   odin_model <- load_odin_model("basic")
   generate_beta_func <- generate_beta
+  seeding_model <- seeding_function
```

Comment:
I think you need to take `seeding_pars` as an input to the models and bind that argument, as with `compare_model`

Reply:
```
basic_model <- function(progression_groups = list(E = 2, asympt = 1, mild = 1, ILI = 1, hosp = 2, ICU = 2, rec = 2),
                        gammas = list(E = 1/(4.59/2), asympt = 1/2.09, mild = 1/2.09, ILI = 1/4, hosp = 2, ICU = 2/5, rec = 2/5),
                        seeding_pars = NULL) {
  model_class <- "sircovid_basic"
  odin_model <- load_odin_model("basic")
  generate_beta_func <- generate_beta
  compare_model <- function(model, pars_obs, data) {compare_output(model, pars_obs, data, type=model_class)}
seeding_model <- function(model, data) {seeding_function(model, data, pars_seeding)}
```

Reply:
I think it's better for future code (e.g. SMC^2) if we dont bind `pars_seeding`